extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -151,4 +196,80 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 		).component();
 	}
 
+	private async handleRegistrationsUpdated(): Promise<void> {
+		const reg = this._controllerModel.getRegistration(ResourceType.sqlManagedInstances, this._miaaModel.namespace, this._miaaModel.name);
+		if (reg) {
+			this._instanceProperties.resourceGroup = reg.resourceGroupName || '-';
+			this._instanceProperties.dataController = this._controllerModel.controllerRegistration?.instanceName || '-';
+			this._instanceProperties.region = (await getAzurecoreApi()).getRegionDisplayName(reg.location);
+			this._instanceProperties.subscriptionId = reg.subscriptionId || '-';
+			this._instanceProperties.computeAndStorage = reg.vCores || '-';
+			this._instanceProperties.host = reg.externalEndpoint || '-';
+			this.refreshDisplayedProperties();
+		}
+	}
+
+	private async handleMiaaStatusUpdated(status: HybridSqlNsNameGetResponse): Promise<void> {
+		this._instanceProperties.status = status.status || '-';
+		this.refreshDisplayedProperties();
+	}
+
+	private handleEndpointsUpdated(endpoints: EndpointModel[]): void {
+		const kibanaQuery = `kubernetes_namespace:"${this._miaaModel.namespace}" and cluster_name:"${this._miaaModel.name}"`;
```

Comment:
>cluster_name [](start = 79, length = 12)

This should be instance_name for sqlmiaa

Reply:
I guess codeflow comments don't render well.  Postgres uses cluster_name but sqlmi uses instance_name

Reply:
Feel free to use Codeflow if you like - doesn't really bother me (fingers crossed that the integration with github is improved at some point...I still really miss it)

Reply:
And I'm not seeing any logs using instance_name at all. But I also don't see any logs for the MIAA instances themselves either regardless...

Reply:
I get some hits for instance_name.  SQL doesn't emit telemetry when it's sitting idle so might have to look back a while, or delete the pod because it emits on startup.  The query looks good to me now.
![image](https://user-images.githubusercontent.com/3500406/84523613-70bca980-ac8d-11ea-802b-820eaa9d3d65.png)